src/Spd.Manager.Licence/BizProfileValidation.cs

```diff
@@ -1,10 +1,14 @@
 using FluentValidation;
+using Microsoft.IdentityModel.Tokens;
+using System.Text.RegularExpressions;
 
 namespace Spd.Manager.Licence;
 public class BizProfileUpdateRequestValidator : AbstractValidator<BizProfileUpdateRequest>
 {
     public BizProfileUpdateRequestValidator()
     {
+        Regex emailRegex = new(@"^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$", RegexOptions.NonBacktracking);
```

Comment:
do not need this. FluentValidation has existing  .EmailAddress() validator

Reply:
I could do that, but it will require creating an additional rule only for that field. So I decided to use regex to put all the rules together for better readability. If you want me to change it anyway, I can definitely do it.

Reply:
Please change it. thx